docs/stdlib/numbers.rst

```diff
@@ -144,7 +144,8 @@ from :eql:type:`str` and :eql:type:`json`.
 
     A 16-bit signed integer.
 
-    An integer value in range from ``-32768`` to ``+32767`` (inclusive).
+    :eql:type:`int16` is capable of representing values from ``-32768`` to
+    ``+32767``, inclusive.
```

Comment:
The format for saying "inclusive" should be consistent across the descriptions here. I suggest that all of them should just use parentheses.
```suggestion
    ``+32767`` (inclusive).
```